src/sensor.h

```diff
@@ -30,6 +30,7 @@ namespace librealsense
     class option;
 
     typedef std::function<void(std::vector<platform::stream_profile>)> on_open;
+    typedef std::function<void(float &val)> on_frame_md;
```

Comment:
The function should receive a reference to `additional_data` and modify its content